File: specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/examples-js/DeleteCertificate.js
const { WebSiteManagementClient } = require("@azure/arm-appservice");
const { DefaultAzureCredential } = require("@azure/identity");
require("dotenv/config");

/**
 * This sample demonstrates how to Description for Delete a certificate.
 *
 * @summary Description for Delete a certificate.
 * x-ms-original-file: specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/examples/DeleteCertificate.json
 */
async function deleteCertificate() {
  const subscriptionId =
    process.env["APPSERVICE_SUBSCRIPTION_ID"] || "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345";
  const resourceGroupName = process.env["APPSERVICE_RESOURCE_GROUP"] || "testrg123";
  const name = "testc6282";
  const credential = new DefaultAzureCredential();
  const client = new WebSiteManagementClient(credential, subscriptionId);
  const result = await client.certificates.delete(resourceGroupName, name);
  console.log(result);
}

File: specification/web/resource-manager/Microsoft.Web/AppService/stable/2025-03-01/examples-java/DeleteCertificate.java

/**
 * Samples for Certificates Delete.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/web/resource-manager/Microsoft.Web/AppService/stable/2025-03-01/examples/DeleteCertificate.json
     */
    /**
     * Sample code: Delete Certificate.
     * 
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void deleteCertificate(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.webApps().manager().serviceClient().getCertificates().deleteWithResponse("testrg123", "testc6282",
            com.azure.core.util.Context.NONE);
    }
}
